### ISSUE TITLE
OSX Docker machine 'x86_64-pc-none' not recognized

### ISSUE THREAD

**Issue by nicoleneo** [MarcAntoine-Arnaud/asdcplib#3]
Might be an issue in OSX or Alpine tag 3.6 being updated? To fix this:

In configure command I added flags `--host=x86_64 --target=x86_64`
```
    && ./configure --build=x86_64 --host=x86_64 --target=x86_64 --enable-phdr --enable-as-02 --with-xerces=/usr/local .. \
```

In build-aux/config.sub I forced $basic_machine to be x86_64-pc.

**Comment by MarcAntoine-Arnaud** [comment]
Is it possible to update and check your path (#4 ) works fine ? and then close it if it's fine.

**Comment by nicoleneo** [comment]
Fixed with my pull request #4
